<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="TYSS-Tanuja-N-Sonawale" failures="1" tests="2" name="Surefire test(failed)" time="85.884" errors="0" timestamp="2024-09-09T13:22:46 IST">
  <testcase classname="testCases.TC02_CreateContact_Test" name="createContacr_Test" time="44.201">
    <failure type="java.lang.AssertionError" message="Contact is created expected [Tanujam] but found [Nagnath]">
      <![CDATA[java.lang.AssertionError: Contact is created expected [Tanujam] but found [Nagnath]
at testCases.TC02_CreateContact_Test.createContacr_Test(TC02_CreateContact_Test.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- createContacr_Test -->
  <testcase classname="testCases.TC01_CreateOrganization_Test" name="viewOrg_Test" time="5.972"/>
  <testcase classname="baseUtility.BaseTest" name="@AfterSuite itsAfterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite itsAfterSuite -->
  <testcase classname="testCases.TC01_CreateOrganization_Test" name="creatingOrg_Test" time="0.0">
    <ignored/>
  </testcase> <!-- creatingOrg_Test -->
  <testcase classname="baseUtility.BaseTest" name="@BeforeSuite itsBeforeSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite itsBeforeSuite -->
</testsuite> <!-- Surefire test(failed) -->
